Kickstarter-iOS/Views/Controllers/ManageViewPledgeRewardReceivedViewController.swift

```diff
@@ -15,4 +47,16 @@ final class ManageViewPledgeRewardReceivedViewController: ToggleViewController {
     _ = self.toggle
       |> checkoutSwitchControlStyle
   }
+
+  // MARK: - View model
+
+  override func bindViewModel() {
+    super.bindViewModel()
+
+    self.viewModel.outputs.rewardReceived
+      .observeForUI()
+      .observeValues { [weak self] isOn in
+        self?.toggle.isOn = isOn
```

Comment:
Didn't you add a RAC binding for this? `self.toggle.rac.on`?

Reply:
I didn't end up using it...let me re-open it and use it for this PR! Will require one more review pass